src/pudl/metadata/resources/eia.py

```diff
@@ -892,27 +892,6 @@
         "sources": ["eia860", "eia923"],
         "etl_group": "outputs",
     },
-    "_out_eia__plants_utilities": {
```

Comment:
Since these tables are no longer going in the DB / Parquet they can be removed from the resource definitions.